### ISSUE TITLE
<hgroup> Element

### ISSUE THREAD

**Issue by jordanwillis** [GIAallen/prj-rev-bwfs-tea-cozy#1]
Everything is looking pretty good from what I have reviewed so far!  One important thing that I wanted to bring up relates to your use of the `<hgroup>` element.  Seeing this tells me you most likely have some prior experience with HTML or perhaps have access to some older resources.  

At any rate, you will want to shy away from using this element in the future as it has officially been removed from the HTML5 specification.  It's important to note that most browsers still understand the tag but there is no guarantee that they will continue to in the future or that other less popular browsers have it implemented at all.

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/hgroup#Usage_notes

The tag was intended to represent a multi-level heading or sub-headings and can only container heading elements inside of it (e.g. `<h1>` thru `<h6>`).  So if you were to use it in the future for some reason (which again I highly discourage), know that it is invalid to put any other type of element inside such.  Also, `<hgroup>` is invalid when used inside of a `<div>` like you have done.

https://validator.w3.org/nu/?doc=https%3A%2F%2Frawgit.com%2FGIAallen%2Fprj-rev-bwfs-tea-cozy%2Fmaster%2FTea%2520Cozy%2Findex.html

So long story short, I would recommend sticking with regular heading elements and using CSS and classes in case you want to style a sub-heading differently.
